src/stimulus-use.ts

```diff
@@ -55,6 +55,10 @@ export class StimulusUse {
     this.logger.groupEnd()
   }
 
+  warn = (functionName: string, args: any): void => {
```

Comment:
I think you can leave out the function name here and remove it from the `warn` function signature. It should anyway be visible from the backtrace itself.